src/input_parsers/tests/Test_VnnLibParser.h

```diff
@@ -0,0 +1,719 @@
+/*********************                                                        */
+/*! \file Test_VnnLibParser.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Idan Refaeli
+ ** This file is part of the Marabou project.
+ ** Copyright (c) 2017-2019 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved. See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** Unit tests for the VnnLibParser class.
+ **/
+
+#include <cxxtest/TestSuite.h>
+
+#include "Engine.h"
+#include "InputQuery.h"
+#include "OnnxParser.h"
+#include "VnnLibParser.h"
+#include <filesystem>
+
```

Comment:
So, all of these are great tests, but they are more regression tests than unit tests.
(I see that the same issue happens with the onnx parser)

Please make this a regression test, and leave the unit test empty for now - and then we can add these tests together, using the CxxTest mock infrastructure.